modules/solid_mechanics/include/kernels/ADDistributedLoadShell.h

```diff
@@ -0,0 +1,79 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
```

Comment:
```suggestion
//* https://mooseframework.inl.gov
```
I have PR currently that is changing all references of .org to .inl.gov

If you would also change every other occurrence in new files, that would be appreciated! Thanks!

Reply:
Easily done with a regex replacement:
```bash
cd moose
git grep -l "www.mooseframework.org" | xargs sed -i "" -e "s|www.mooseframework.org|mooseframework.inl.gov|g"
```

Reply:
@milljm thanks for the hint! I assume I need to wait until your PR is merged and then update all the references. Otherwise, I'll have to commit thousands of changes. Is that correct?

Reply:
No, just the files you're adding!